kolibri/plugins/device/assets/src/views/FacilitiesPage/index.vue

```diff
@@ -3,14 +3,17 @@
   <div>
     <HeaderWithOptions :headerText="coreString('facilitiesLabel')">
       <template #options>
+        <!-- Margins to and bottom adds space when buttons are vertically stacked -->
         <KButtonGroup>
           <KButton
             :text="$tr('syncAllAction')"
+            style="margin-top: 16px; margin-bottom: -16px;"
```

Comment:
added example of how this could be addressed globally here

https://github.com/learningequality/kolibri-design-system/pull/290